yarn.lock

```diff
@@ -3,88 +3,81 @@
 
 
 "@ampproject/toolbox-core@^2.6.0", "@ampproject/toolbox-core@^2.7.1":
-  version "2.7.1"
-  resolved "https://registry.yarnpkg.com/@ampproject/toolbox-core/-/toolbox-core-2.7.1.tgz#d433a333c0dbb0bb6db7e69edd490d5694e06fc3"
-  integrity sha512-MWGmCLyBOouXTy1Vc30Jw7NkshJ5XkPlcXhhRc9Gw3dDAZJ8rUS69SIQ6cFMt2owCQnw7irMNlvZQTqdyx61rA==
+  "integrity" "sha512-MWGmCLyBOouXTy1Vc30Jw7NkshJ5XkPlcXhhRc9Gw3dDAZJ8rUS69SIQ6cFMt2owCQnw7irMNlvZQTqdyx61rA=="
+  "resolved" "https://registry.npmjs.org/@ampproject/toolbox-core/-/toolbox-core-2.7.1.tgz"
```

Comment:
This is probably because your setup is different. Check if it is because of using `npm` instead of `yarn` or some specific configuration in `yarn`.

I would suggest that you restore this file to the version in `master` and run `yarn` again to get an updated file. If this isn't working, then it is better to not include any changes to `yarn.lock`. I have to update the file in `master` anyway after merging with other changes.

Reply:
Yes, I used `npm` instead of `yarn`. I will try to quickly fix it now. 🙌

Reply:
@sarathms I have reverted the file to the master version. It should be good to go!